php/Terminus/Dispatcher/Subcommand.php

```diff
@@ -177,7 +177,12 @@ private function validate_args( $args, $assoc_args, $extra_args ) {
       exit(1);
     }
 
-    $unknown_positionals = $validator->unknown_positionals( $args );
+    $invalid = $validator->invalid_positionals($args);
+    if($invalid) {
+      \Terminus::error("Invalid positional value: $invalid");
```

Comment:
What is the '\' notation? We don't use that in `php/class-terminus-command.php`, should they be consistent?